apps/benchmark/mali_gpu_imagenet_bench.py

```diff
@@ -0,0 +1,90 @@
+"""Benchmark script for ARM CPU.
+see README.md for the usage and results of this script.
```

Comment:
README hasn't been updated for this script. If the usage is same as the arm cpu bench script, just write it so.